keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -257,30 +236,27 @@ spec:
     name:
     namespace:
   protocol: [10]
-  protocolAzureBlob: [11]
-    storageAccountName:
-    accountKey:
-    containerName:
-  protocolS3: [12]
-    endpoint:
-    accessKeyId:
-    bucketName:
-    region:
-    signatureVersion:
-    userName:
-  protocolGcs: [13]
-    bucketName:
-    privateKeyName:
-    projectId:
-    serviceAccount:
+    type: ""
```

Comment:
comment re. `SecretRef` above:  1) should this be named _provisionerSecretRef_ for bucket class consistency? 2) Does this ref also need the _uuid_ and _resourceVersion_ fields as shown for `bucketRef`?